awesome-books-with-ES6/modules/DummyData.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-cycle */
+import Storage from './Storage.js';
+import { bookContainer } from '../index.js';
```

Comment:
- [ ] Linters help us write clean and maintainable code. Kindly enable linters here and resolve the resulting error. You are getting this `import/no-cycle` error because you are importing bookContainer from **`index.js`** (a file where you are also importing **`DummyData.js`**). You could resolve this by exporting the `bookContainer` declaration from `index.js` into a new (different) file and then importing bookContainer here in `DummyData.js`. You could follow this pattern for the same linter error.